internal/ingress/controller/store/store.go

```diff
@@ -598,6 +598,9 @@ func (s k8sStore) ListIngresses() []*extensions.Ingress {
 			continue
 		}
 		for ri, rule := range ing.Spec.Rules {
+			if rule.HTTP == nil {
+				continue
+			}
```

Comment:
Is there no valid case for the ingress to not have any `rule.HTTP` defined? I see that below we create a default path when it does not exist, why do we not do the similar thing for rule.HTTP?

Reply:
@ElvinEfendi when you only define a backend in the ingress without rules. This acts as `/`

Reply:
Are you referring to something like
```
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: test-ingress
spec:
  backend:
    serviceName: testsvc
    servicePort: 80
```
? If so then in the case `ing.Spec.Rules` would be `nil` too no?

Reply:
Yes